src/views/connectionUI.ts

```diff
@@ -674,24 +674,19 @@ export class ConnectionUI {
 	}
 
 	// Prompts the user to pick a profile for removal, then removes from the global saved state
-	public removeProfile(): Promise<boolean> {
+	public async removeProfile(): Promise<boolean> {
 		let self = this;
```

Comment:
While you're looking at this stuff - most of these self = this calls can be removed. That was a pattern required when using function variable callbacks but nowadays it's much simpler to just use lambda (arrow) functions which capture this without needing to do the self=this. So I like to remove this whenever I come across it in code I'm modifying.